src/component/inspect.ts

```diff
@@ -1,12 +1,12 @@
 import { v } from "convex/values";
 import { Id } from "./_generated/dataModel.js";
 import { DatabaseReader, query } from "./_generated/server.js";
-import { getTree, p } from "./btree.js";
+import { getTree, Namespace, p } from "./btree.js";
 
 export const display = query({
```

Comment:
wow I've missed the existence of this - would be nice to document how to view / debug your state in the readme. One common thing for me is trying to figure out why some documents ended up busted - and finding myself crawling through the dashboard data to try to look for the data ordering. another utility I've wanted is for table aggregate: "refresh this document with the current doc value / sort key" - and a backfill utility to just crawl a table and re-calculate every doc would be handy in dev. maybe just copying from your example is enough there though